packages/trackers/src/analytics/facebook.pixel.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Core Packages
+ */
+import injector, { js } from '@ingresse/injector';
+
+/**
+ * Local values
+ */
+const prefix = 'trackers:fbq:';
+
+/**
+ * Facebook Pixel
+ */
+let fbq = {
+    key: '',
+};
+
+/**
+ * Library status verification
+ *
+ * @returns {boolean}
+ */
+fbq.running = function() {
+    try {
+        return (typeof window.fbq === 'function');
+    } catch (e) {
+        return false;
+    }
+}
+
+/**
+ * Initializer
+ *
+ * @param {object|string} optionsOrKey
+ *
+ * @param {object}
+ */
+fbq.init = function(optionsOrKey) {
+    return new Promise((resolve, reject) => {
+        let options = {
+            key    : '',
+            id     : 'fbq-js',
+            target : 'head',
+            delay  : 5000,
+            version: '2.0',
+        };
+
+        if (optionsOrKey) {
+            if (typeof optionsOrKey === 'string') {
+                options.key = optionsOrKey;
+            }
+
+            if (typeof optionsOrKey === 'object') {
+                options = {
+                    ...options,
+                    ...optionsOrKey,
+                };
+            }
+        }
+
+        const {
+            key,
+            id,
+            target,
+            delay,
+            version,
+        } = (options || {});
+
+        if (!key) {
+            return reject({
+                code   : -1,
+                message: `${prefix}missing-key-param`,
+            });
+        }
+
+        if (key && (key === fbq.key)) {
+            return resolve();
+        }
+
+        const content = `!function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='${version}';n.queue=[];t=b.createElement(e);t.async=!0;t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window, document,'script','https://connect.facebook.net/en_US/fbevents.js');fbq('init', '${key}');fbq('track', 'PageView');`;
```

Comment:
is this a good idea? Is not a better way to inject from the cdn? Do you think that have any downside?

Reply:
By the developer's documentation, that's the way:
- https://developers.facebook.com/docs/facebook-pixel/implementation

Reply:
cool